jaeger-zipkin/build.gradle

```diff
@@ -1,12 +1,23 @@
 description = 'Integration library for Zipkin'
 
 dependencies {
-    compile group: 'io.zipkin.reporter', name: 'zipkin-reporter', version: '1.1.2'
-    compile group: 'io.zipkin.reporter', name: 'zipkin-sender-urlconnection', version: '1.1.2'
+    compile(group: 'io.zipkin.reporter', name: 'zipkin-reporter', version: '1.1.2') {
+        exclude group:'io.zipkin.zipkin2'
+        exclude group:'io.zipkin.java'
+    }
+    compile(group: 'io.zipkin.reporter', name: 'zipkin-sender-urlconnection', version: '1.1.2') {
+        exclude group:'io.zipkin.zipkin2'
```

Comment:
Is this needed, if you are declaring it as a direct dependency of your project?

Reply:
Tested it, and you are right, it doesn't appear to be necessary due to the explicit dependency.